index.html

```diff
@@ -1086,6 +1092,120 @@ <h3 id="applying">
       </p>
       <section>
         <h3>
+          <code>dir</code> member
+        </h3>
+        <p>
+          The <dfn><code>dir</code> member</dfn> specifies the <dfn>default
```

Comment:
I would remove the word "default" from "default base direction". The base direction is the starting paragraph direction for the Unicode bidi algorithms (UAX#9, see BD5).